infra/config/example.ts

```diff
@@ -0,0 +1,12 @@
+import { EnvConfig } from "../lib/env-config";
+import { EnvType } from "../lib/env-type";
+
+export const config: EnvConfig = {
+  environmentType: EnvType.production,
+  region: "us-east-1",
+  vpcId: "my_vpcId",
+  dbName: "my_db",
+  dbUsername: "my_db_user",
+  fhirServerECRName: "fhir-server-ecr-name",
+};
```

Comment:
assuming there'll be another PR to put the actual config in the internal repo?

Reply:
yep! https://github.com/metriport/metriport-internal/pull/371